client/src/web3dice.js

```diff
@@ -72,32 +74,44 @@ export const web3dice = {
     }
   },
 
-  getBlock() {
-    this.provider.getBlockNumber().then(block => {
-      store.block = block
-      console.log('block number is' + block)
-    })
+  async delayedRoll(diceId) {
+    let rollDelay = Math.floor(Math.random() * 100)
+    console.log('delay', rollDelay)
+    setTimeout(() => {this.roll(diceId)}, rollDelay)
   },
+
   async roll(diceId) {
+    store.isRolling[diceId] = true
     console.log('rolling')
     try {
       const roll = await this.diceContract.roll(diceId)
       store.lastRoll[diceId] = roll
       console.log('roll', roll)
+      //store.isRolling[diceId] = false
+      setTimeout(() => {store.isRolling[diceId] = false}, 1000)
       return roll
     } catch (err) {
       console.log("Error: ", err)
+      store.isRolling[diceId] = false
       return 'failed'
     }
   },
+
   async getTraits(diceId) {
     try {
       const traits = await this.diceContract.getTraits(diceId)
+      store.diceTraits[diceId] = traits
+      return traits
     } catch (err) {
       console.log("Error: ", err)
       return 'failed'
     }
-    store.diceTraits[diceId] = traits
-    return traits
+  },
+
+  async getOwnedDice() {
+    const owned = [1,2,1,2,1,2]
```

Comment:
Is this just a dummy entry?

Reply:
Yes, until I can pull a real list of owned items.